src/garage/torch/algos/vpg.py

```diff
@@ -11,6 +11,22 @@
 from garage.torch.algos import loss_function_utils as loss_utils
 
 
+def _add_padding_2d(vals, length):
+    tensor = torch.Tensor(vals)
+    padding_length = max(length - tensor.shape[0], 0)
+    return F.pad(tensor, (0, 0, 0, padding_length))
+
+
+def _add_padding_1d(vals, length):
+    tensor = torch.Tensor(vals)
+    padding_length = max(length - tensor.shape[-1], 0)
+    return F.pad(tensor, (0, padding_length))
```

Comment:
please add these to a new module called `path_utils` with full docstrings and tests

how do the 1d and 2d case differ?

Reply:
created `path_util.py` and moved the functions
made `pad_to_last` new funciton and remove `_add_padding_1d` and `_add_padding_2d`